src/d3a/d3a_core/cli.py

```diff
@@ -98,11 +98,14 @@ def main(log_level):
 @click.option('--enable-bc', is_flag=True, default=False, help="Run simulation on Blockchain")
 @click.option('--compare-alt-pricing', is_flag=True, default=False,
               help="Compare alternative pricing schemes")
+@click.option('--enable-external-connection', is_flag=True, default=False,
+              help="External Agents interaction to simulation during runtime")
```

Comment:
having trouble understanding the need for this flag. We lived without it for now. I vote to omit it.

Reply:
I added it since it opens redis communication that runs in thread and slightly slowsdown the simulation which could be avoided for cases when external communication is not needed.

Reply:
Yes, ok, understood. Although speed is not an issue locally. 
Leave it here though.